tokens/README.md

```diff
@@ -1,23 +1,19 @@
-Copies the authentication tokens for Hail Batch service accounts from
-Kubernetes secrets to the Secret Manager. There it is accessible by the server
-that launches Batch pipelines on behalf of users.
+# Copying Hail tokens
```

Comment:
The idea was that the server component runs a pipeline on your behalf, using the token that corresponds to the Hail Batch service account for a particular project.

Turns out we can't actually use it the way it was intended originally at all, because one can't check group memberships with a non-Enterprise edition of Google Workspace :(.

So this will likely be gone soon -- still working on the server part.

Reply:
Thanks!